### PR TITLE
Update Prometheus, Node Exporter & Grafana

### PR DESCRIPTION
**Pull request by rajatvig**

### CODE REVIEW

File: jsonnet/kube-prometheus/prometheus/prometheus.libsonnet [hunk 1]
```diff
@@ -6,7 +6,7 @@ local k3 = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
     namespace: 'default',
 
     versions+:: {
-      prometheus: 'v2.7.2',
+      prometheus: 'v2.9.2',
```
Comment (brancz) [inline jsonnet/kube-prometheus/prometheus/prometheus.libsonnet]: v2.10.0 is already out?

Comment (rajatvig) [reply jsonnet/kube-prometheus/prometheus/prometheus.libsonnet]: Will update and run the regenerate

**Comment by rajatvig** [comment]
@brancz Have updated the generated manifests. There is an issue though as the repository depends on itself, the lock file changes which I have omitted. It would likely need a bump once this merges to master.

**Comment by brancz** [comment]
lgtm thanks!

**Pull request merged**